tools/qr-generator/src/generator/qr.js

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs'
+import puppeteer from 'puppeteer'
+import Mime from 'mime'
+import { padNumberToSameLength } from '../util.js'
+import * as path from 'path'
+import ProgressBar from 'progress'
+
+const generateQrCode = async (page, { token, frontendUrl, qrImage, qrCodeStyling }) => {
+  const data = `${frontendUrl}/token/scan?token=${token.token}`
+  const renderOptions = { ...qrCodeStyling, data, image: qrImage }
+  const result = await page.evaluate((renderOptions) => renderQrCode(renderOptions), renderOptions)
+  if (!result.successful) {
+    console.error(result.error)
+    throw new Error('QR kód generálása sikertelen!')
+  }
+  return result
+}
+
+export const generateQrCodes = async (tokens, args) => {
+  const { count, qrOutDirectory, frontendUrl, qrCodeStyling, qrRendererTemplate, qrImage } = args
+  const bar = new ProgressBar(':bar :current/:total :percent', { total: count })
+  const browser = await puppeteer.launch({ headless: true })
```

Comment:
The qr styling library is only for browsers, so I had to do it this way

Reply:
Ez a fork elvileg működik node-dal: https://github.com/Loskir/styled-qr-code

Reply:
Vagy CLI helyett mehetne a frontendbe

Reply:
Az nagyon overkill lenne